### DISCUSSION TITLE
[Request] 可以设置所有人都能看到并使用的默认助手吗？

### DISCUSSION THREAD

**Discussion by hackerYHT** [lobehub/lobehub#4094] [General Question]
### 🥰 需求描述

 可以设置所有人都能看到并使用的默认助手吗？自己想通过langchain搭建一个RAG助手，接入lobe-chat前端进行使用，作为默认的RAG助手给所有人使用，别人不能删除修改的那种

### 🧐 解决方案

 可以设置所有人都能看到并使用的默认助手吗？

[a comment by an automated account is not shown]

**Comment by mujiannan** [comment]
Duplicated with #3650

**Comment by ZeroDeng01** [comment]
有个思路就是你先构建一个RAG插件，然后环境变量里面有默认助手设置，在默认助手设置里面配置使用你的插件，这样所有用户新建默认助手就是你环境变量配置的内容了

[a comment by an automated account is not shown]

**Comment by hackerYHT** [comment]
> 有个思路就是你先构建一个RAG插件，然后环境变量里面有默认助手设置，在默认助手设置里面配置使用你的插件，这样所有用户新建默认助手就是你环境变量配置的内容了

非常感谢！这也是个非常好的思路。但是如果同时想能够自定义多套默认RAG-Agent提供给别人使用好像比较困难

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment]
@hackerYHT 可以做一个插件，然后一个api 一个 agent
